lib/helpers/rule-helper.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @fileoverview Base for all rules
+ */
+'use strict';
+
+function RuleHelper(context) {
+   var self = this;
+
+   function getIndent() {
+      if (context.options.length > 0) {
```

Comment:
The option to configure the rules was added, and thus needs to be documented. Also, please make the configuration option an object (rather than an array of option values) so that it's easier to read the configuration. The default config object should be:

```
{
   'IndentChar': 'space',
   'IndentAmount': 3
}
```